awscli/customizations/s3/filegenerator.py

```diff
@@ -173,6 +173,12 @@ def list_files(self, path, dir_op):
         if not self.should_ignore_file(path):
             if not dir_op:
                 size, last_update = get_file_stat(path)
+                if last_update is EPOCH_TIME:
```

Comment:
Ok, that makes sense.  I think using an identity instead of an equality check though is very subtle.  I'd prefer a more explicit sentinel of just "we couldn't figure out the time" and let the consuming code figure out what they want to do in those scenarios.